src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
```

Comment:
Consider using constants for magic numbers. In this case, the number 2 used in the area calculation can be defined as a constant to give it a meaningful name and to avoid magic numbers in the code.